packages/strapi-design-system/src/Tooltip/Tooltip.js

```diff
@@ -16,8 +16,8 @@ const TooltipWrapper = styled(Box)`
 `;
 
 export const Tooltip = ({ children, label, description, delay, position, id, ...props }) => {
-  const tooltipId = useId('tooltip', id);
-  const descriptionId = useId('description');
+  const tooltipId = useId(id);
+  const descriptionId = useId(id);
```

Comment:
wouldn't this mean the tooltipID & descriptionIds are equal because we just pass back the `initialId`?

Reply:
Would it be possible to move away from `initialId`? It isn't supported in react@18 neither and I find it quite confusing :)

Reply:
I'll check for the tooltip @joshuaellis 
I don't know if we should @gu-stav - I don't know potential consequences (people using their own ids when using our inputs), I didn't want to make the call and was not sure it needed to be done now 🤷🏻‍♀️

Reply:
> Would it be possible to move away from initialId? It isn't supported in react@18 neither and I find it quite confusing :)

Personally, i agree. _However_, it would break people's tests if they're looking for ids (imo bad practice though) and it would break if people use `ids` in the code – also imo bad practice not a react pattern. Do we want to deal with the potential fallout? I mean we can already see that it "broke" our tests already 😂

Reply:
good catch @joshuaellis thanks! fixed [here](https://github.com/strapi/design-system/pull/853/commits/1e39e04569213ede53be27efaf4b12ef0dace41c)